backend/routes/postReaction.py

```
@@ -25,207 +25,180 @@ def get_db():
+    return {
+        "id": reply.id,
+        "content": reply.content,
+        "created_at": reply.created_at,
+        "user": serialize_user(reply.user),
+        "total_likes": len(reply.likes),
+        "user_liked": any(l.user_id == user.id for l in reply.likes)
+    }
 
 
-#delete the comment
+def serialize_user(user: User):
```

Ruff (0.8.2)

Missing return type annotation for public function `serialize_user`

(ANN201)